app/code/Magento/GoogleGtag/view/frontend/templates/head.phtml

```diff
@@ -0,0 +1,35 @@
+<?php
+/**
+ * Copyright © Magento, Inc. All rights reserved.
+ * See COPYING.txt for license details.
+ */
+?>
+<?php
+/** @var \Magento\Framework\View\Element\Template $block */
+/** @var \Magento\Framework\Escaper $escaper */
+/** @var \Magento\GoogleGtag\ViewModel\Adwords $adsViewModel */
+$adsViewModel = $block->getViewModel();
+?>
+
+<?php if ($adsViewModel->isGoogleAdwordsConfigurable() && $adsViewModel->isGoogleAdwordsActive()): ?>
+    <?php $conversionId = $adsViewModel->getConversionId(); ?>
+    <?php $gtagSiteSrc = $adsViewModel->getConversionGtagGlobalSiteTagSrc(); ?>
+    <script>
```

Comment:
Let's see how can we avoid inline javascript

Reply:
Moved js in separate file